lib/convey/handler/ChannelHandlers.tsx

```diff
@@ -14,34 +13,21 @@ const notification = (message: any, channelId = "") => {
     }
 };
 
-const log = (message: any, channelId = "") => {
-    console.debug(channelId, "Log:");
-    console.debug(message);
-};
-
-const text = (message: TextMessage) => {
-    if (message.elements) {
-        CustomRenderer.render(message);
-    }
-
-    console.debug("Text:");
-    console.debug(message);
+const logging = (message: any, channelId = "") => {
+    console.debug(channelId, "Logging:", message);
 };
 
 const interaction = (message: any, channelId = "") => {
-    console.debug(channelId, "Interaction:");
-    console.debug(message);
+    console.debug(channelId, "Interaction:", message);
```

Comment:
if (message.elements) {
        CustomRenderer.render(message);
    }

interaction sollte das trotzdem aufrufen. 

"text" wurde lediglich in "interaction" umbenannt